obip-0008.md

```diff
@@ -0,0 +1,146 @@
+<pre>
+OBIP: 8
+Title: 3rd Party Moderator Verification
+Author: Tyler Smith <tyler@ob1.io>
+Discussions-To: Github Issues
+Status: Draft
+Type: Standards Track
+Created: 02/28/2018
+Copyright: MIT
+</pre>
+
+## Abstract
+A specification for a system allowing 3rd parties to provide a list of moderators that it has verified in some way, with the nature and extent of the verifications being defined by the 3rd party itself.
+
+## Motivation
+Allow buyers on OpenBazaar to have more information about the identity and reputation of moderators for given listings. If a 3rd party can provide accurate verifications of the moderator's identity, knowledge, fairness, and/or reliability then users can make safer decisions.
+
+## Verified Moderators Endpoint
+
+To provide verified moderators a provider must have a public endpoint returning data with the following schema:
+
+#### Example
+
+```json
+GET https://bazaarco.example.com/verified_moderators
+
+{
+  "data": {
+    "name": "BazaarCo",
+    "description": "Moderators verified by BazaarCo",
+    "link": "https://bazaarco.example.com/verifications"
+  },
+  "types": [
+    {
+      "name": "standard",
+      "description": "A moderator that has been vetted by BazaarCo",
+      "badge": "https://bazaarco.example.com/standard_badge.png"
+    }{
+      "name": "small_bonded",
+      "description": "A moderator that has bonded $1000 with BazaarCo",
+      "badge": "https://bazaarco.example.com/small_bonded_badge.png"
+    }
+  ],
+  "moderators": [
+    {
+      "peerID": "QmXFMkpBBpL4zcYAArVAecLyypFrRzp2Co4q9oXUtzF7XF",
+      "type": "standard"
+    },
+    {
+      "peerID": "QmVFNEj1rv2d3ZqSwhQZW2KT4zsext4cAMsTZRt5dAQqFJ",
+      "type": "small_bonded"
+    }
+  ]
+}
+```
+
+#### Response Explanation
+
+- `data` contains general overview information about the service to be displayed to the user in the client.
+  - `name` is the name of the verification service.
+  - `description` is a brief explanation of what this service is offering.
+  - `link` is a URI to a document explaining further information about the service.
+- `types` is an array detailing the types of verifications being offered.
+  - `name` is the name of the verification type.
+  - `description` is a brief explanation of what this type represents.
+  - `badge` is a URI pointing to a badge icon to display on verified listings.
```

Comment:
The badge graphic is expected to be at least 100px x 100px, and have a square aspect ratio.

Should we add a set of images, like we do with products, so clients can display optimal sized images in different situations, and handle hi-resolution devices?

ie:
'badge': {
  'tiny': uri, (60x60)
  'small': uri, (120x120)
  'medium': uri, (240x240)
  'large': uri (480x480) 
}

The above are the same sizes used for avatars.